moment_kinetics/src/gauss_legendre.jl

```diff
@@ -906,6 +913,26 @@ function setup_global_weak_form_matrix!(QQ_global::Array{mk_float,2},
             QQ_global[end,:] .= 0.0
             QQ_global[end,end] = 1.0
         end
+        # requires RHS vector b[1],b[end] = boundary values
+    end
+    if periodic_bc
+        # Make periodic boundary condition by modifying elements of matrix for duplicate point
+        # add assembly contribution to lower endpoint from upper endpoint
+        j = coord.nelement_local
+        get_QQ_local!(QQ_j,j,lobatto,radau,coord,option)
+        iminl = imin[j] - (coord.nelement_local > 1)

```

Comment:
This line looks illegal. Can we use another line instead of this Julia feature (where presumably Bool => int?)

Reply:
Or an explicit type conversion to make it clear what is going on?

Reply:
Whatever you like, I don't mind.